client-app/core/types/theme-config.ts

```diff
@@ -4,6 +4,8 @@ import type { ISocialSharingService } from "./social-sharing";
 import type { SortDirection } from "@/core/enums";
 
 export interface IThemeConfigPreset {
+  password_expiry_in_days?: number;
```

Comment:
XAPI already has setting on how long user should notified before password gets expired. You don't need to duplicate it
https://github.com/VirtoCommerce/vc-module-profile-experience-api/blob/5fb732ad06dfc3f6b4203ee08b38aa9b4092ced7/src/VirtoCommerce.ProfileExperienceApiModule.Data/Schemas/UserType.cs#L99